adblock-lean

```diff
@@ -26,6 +26,16 @@ adblock-lean custom commands:
 	gen_config	generate default config
 	update		update adblock-lean to the latest version"
 
+mkdir -p /var/run/adblock-lean
+
+trap cleanup INT TERM EXIT
+
+cleanup()
+{
+	log_msg "Cleaning up and exiting adblock-lean."
+	rm -rf /var/run/adblock-lean
```

Comment:
Why not clean all temp files here as well? When OOM crashes occur, some of them don't get cleaned up.

Reply:
Doesn’t removing the directory remove all the temporary files?

Reply:
My bad, I didn't realize this was a directory.